api/v1alpha1/toolchaincluster_types.go

```diff
@@ -66,6 +79,13 @@ type LocalSecretReference struct {
 // cluster updated periodically by cluster controller.
 // +k8s:openapi-gen=true
 type ToolchainClusterStatus struct {
+	// APIEndpoint is the API endpoint of the remote cluster. This can be a hostname,
+	// hostname:port, IP or IP:port.
+	APIEndpoint string `json:"apiEndpoint,omitempty"`
+
+	// OperatorNamespace is the namespace in which the operator runs in the remote cluster
+	OperatorNamespace string `json:"operatorNamespace,omitempty"`
```

Comment:
minor - but usually we set `omitempty` with optional fields only.
```suggestion
	// APIEndpoint is the API endpoint of the remote cluster. This can be a hostname,
	// hostname:port, IP or IP:port.
	APIEndpoint string `json:"apiEndpoint"`

	// OperatorNamespace is the namespace in which the operator runs in the remote cluster
	OperatorNamespace string `json:"operatorNamespace"`
```